### DISCUSSION TITLE
欢迎来到WTF科学家社区!

### DISCUSSION THREAD

**Discussion by AmazingAng** [AmazingAng/WTF-Solidity#1] [Announcements]
## 👋 欢迎来到WTF科学家社区!
我们将Github Discussion用作与社区其他成员联系的地方。 我们希望您：
   * 问你想知道的问题。
   * 分享意见。
   * 与其他社区成员互动。
   * 欢迎他人，心胸开阔。 请记住，这是我们的社区一起建造💪。
  
 请在下面发表评论并自我介绍，并告诉我们你希望WTF科学家社区能帮助你什么。

**Comment by metaforcode** [comment]
热烈欢迎&占位

**Reply by chandlerniu** [reply]
people坚定持有者来报道啦！

**Reply by beihaili** [reply]
wgmi

**Reply by qfengthree** [reply]
占个位子

**Reply by jt603** [reply]
新人报道

**Comment by Azaleine** [comment]
来咯

**Comment by FFFFourwood** [comment]
板凳

**Comment by tzzydh** [comment]
占位

**Comment by dtwangquan** [comment]
占位

**Comment by Sein-Ding** [comment]
来咯

**Comment by liutongxin** [comment]
Hi, 我是summer，现在是一个Java开发，正在转型Web3，学习Solidity中，希望和社区一起成长，自己学习的同时也能帮到大家

**Reply by simpledavid** [reply]
我是java 和C++开发

**Comment by JustinAsdz** [comment]
Hi This is Asdz

**Comment by dragon2031** [comment]
Coming！

**Comment by yj666** [comment]
学习！

**Comment by xuzipeng** [comment]
来了来了 。

**Comment by iamlujue** [comment]
来了来了

**Comment by 567568** [comment]
占座

**Comment by mst1995** [comment]
占座

**Comment by githublister** [comment]
- - - - - - - -- - -&nbsp;求最新财富密码




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年4月24日(星期天) 下午5:11
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; 
主题: Re: [AmazingAng/WTFSolidity] 欢迎来到WTF科学家社区! (Discussion #1)





 
占座

**Reply by makeyang-spc** [reply]
微信群号是多少啊？

**Comment by AzumWatson** [comment]
这是好的

**Comment by 0xmi** [comment]
观摩一下

**Comment by chad779** [comment]
像个论坛.

**Comment by 0xmi** [comment]
哈喽科学家

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Rulesbreaker** [comment]
在5. 引用类型, array, struct中，提到“bytes比较特殊，是数组，但是不用加[]”，然后在对应的代码中有写到如下两行：
`bytes1[5] array2;  //line 7`
`bytes1[] array5; //line 12`

请问这里的bytes1是指一个字节么，如果是那数组是否应该是定长的？不知道问题是否能发在discussion，如果不合适我就删掉，谢谢！

**Reply by AmazingAng** [reply]
可以这么理解 bytes32就是bytes1[32]
bytes32[5]是bytes32的定长数组
bytes32[]是bytes32的不定长数组

**Reply by Rulesbreaker** [reply]
明白了，谢谢！

**Comment by wangluxu** [comment]
站位

**Comment by jackma0077** [comment]
我来了

**Comment by gofenix** [comment]
来了

**Comment by 0xkeenz** [comment]
支持~

**Comment by Augly** [comment]
学习

**Comment by CadeYu** [comment]
hi

**Comment by rensx5514** [comment]
Hello boys and girls.

**Comment by jackma0077** [comment]
hello



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2022年10月31日(周一) 晚上8:00
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [AmazingAng/WTF-Solidity] 欢迎来到WTF科学家社区! (Discussion #1)




 
Hello boys and girls.

**Comment by SimonsJ4r** [comment]
Hi, 我是 0xSimon, 加入 WTF 学习了

**Comment by xinchen37** [comment]
支持

**Comment by mj5219054** [comment]
热烈欢迎&占位

**Comment by liwuhou** [comment]
火钳留名

**Comment by EthanOK** [comment]
hello everyone!

**Comment by luyao-pilgrim** [comment]
start my solidity journey, thank you!

**Comment by qfengthree** [comment]
小白学习

**Comment by justirycn** [comment]
学习完 打卡

**Comment by qfengthree** [comment]
怎么打卡？









&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "AmazingAng/WTF-Solidity"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年5月17日(星期三) 晚上6:24
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [AmazingAng/WTF-Solidity] 欢迎来到WTF科学家社区! (Discussion #1)





 
学习完 打卡

**Comment by zcddtb** [comment]
come here

**Comment by 637leaf** [comment]
您好在学习wtf的教程 是在哪里能看到我做错的题呢

**Comment by nft-maker-one** [comment]
希望AA大佬翻个牌，想在学校给同学们讲讲solidity教程，自己能力确实不够，再加上WTF的教程真的太棒了，所以就偷了个懒，这个视频录制下来后可以上传到互联网

**Reply by nft-maker-one** [reply]
不好意思忘了加个吗！！！！哈哈，可以上传到互联网吗

**Reply by AmazingAng** [reply]
可以的，欢迎使用！

**Reply by AmazingAng** [reply]
视频记得说下用的wtf solidity教程，做好署名

**Reply by nft-maker-one** [reply]
谢谢aa哥，署名这是必须的。哈哈哈，aa哥没收个版权费我就很感激了(〃'▽'〃)(〃'▽'〃)

**Comment by jackma0077** [comment]
你好，很高兴来到WTF



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年7月25日(周二) 晚上10:12
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [AmazingAng/WTF-Solidity] 欢迎来到WTF科学家社区! (Discussion #1)




 
可以的，欢迎使用！

**Reply by HungYann** [reply]
占个位置

**Comment by sqhEth** [comment]
持续学习ing

**Comment by 195286381** [comment]
通过WTF进入WEB3领域😄, 社区加油⛽️

**Comment by Macin20** [comment]
宝藏社区啊！

**Comment by Swiftevo** [comment]
你好，我是 DeSci Asia 的 Co founder Swift Evo .

很高興看到 WTF 開設了 DeSci 的社區

**Comment by nft-maker-one** [comment]
大家好，我是Jerry，现在大四即将毕业，现在还在传统web2开发实习，但在个人的账户上出了许多关于web3基础知识和程序开发的视频文档。希望可以和大家互相学习，一起交流行业最新热点，最新技术

**Comment by makeyang-spc** [comment]
发个链接参观一下可以吗



在 2024-11-17 15:56:25，"web3摸鱼者" ***@***.***> 写道：

大家好，我是Jerry，现在大四即将毕业，现在还在传统web2开发实习，但在个人的账户上出了许多关于web3基础知识和程序开发的视频文档。希望可以和大家互相学习，一起交流行业最新热点，最新技术

**Comment by HtpsDeathSky** [comment]
Solidity101第13章继承在讲述菱形继承的时候是不是有点问题， 菱形继承示例中会同时打印adam.bar和eve.bar，我把代码复制部署，只会打印adam.bar和god.bar，菱形继承的情况下，父类有同样方法不应该只会执行其中一个么，solidity新手哈QAQ
